images/vep/Dockerfile

```diff
@@ -45,7 +47,7 @@ RUN rm $VEP_SHARE/TissueExpression.pm \
   $VEP_SHARE/svm.pl \
   $VEP_SHARE/utr_splice.pl && \
   git clone https://github.com/populationgenomics/loftee_38.git && \
-    cp -r loftee_38/* $MAMBA_ROOT_PREFIX/share/ensembl-vep-$VERSION-1 \
+    cp -r loftee_38/* $VEP_SHARE \
     && rm -rf loftee_38
```

Comment:
trying this out locally now, though I didn't think there was any way in Docker to persist values from RUN into the environment

Reply:
This didn't work unfortunately - the ENV variable isn't persisted to the image. This can be set at container runtime, otherwise we'll need a way to parametrize the ENV command

Reply:
We might have to append the `export` to `.bashrc` for this to work.